src/GitLink/Linker.cs

```diff
@@ -131,5 +162,41 @@ public static bool Link(string pdbPath, LinkOptions options = default(LinkOption
 
             return true;
         }
+
+        private static void CreateSrcSrv(string srcsrvFile, SrcSrvContext srcSrvContext)
+        {
+            Argument.IsNotNull(() => srcSrvContext);
+            Argument.IsNotNullOrWhitespace(() => srcSrvContext.RawUrl);
+            Argument.IsNotNullOrWhitespace(() => srcSrvContext.Revision);
+            Argument.IsNotNullOrWhitespace(() => srcsrvFile);
+
+            if (srcSrvContext.VstsData.Count != 0)
+            {
+                File.WriteAllBytes(srcsrvFile, SrcSrv.CreateVsts(srcSrvContext.Revision, srcSrvContext.Paths, srcSrvContext.VstsData));
+            }
+            else
+            {
+                File.WriteAllBytes(srcsrvFile, SrcSrv.Create(srcSrvContext.RawUrl, srcSrvContext.Revision, srcSrvContext.Paths, srcSrvContext.DownloadWithPowershell));
+            }
+        }
+
+        private static string GetNormalizedPath(string path, string gitRepoRootDir)
+        {
+            Argument.IsNotNullOrEmpty(nameof(path), path);
+            Argument.IsNotNullOrEmpty(nameof(gitRepoRootDir), gitRepoRootDir);
+
+            string relativePath = Catel.IO.Path.GetRelativePath(path, gitRepoRootDir);
+            string[] segments = relativePath.Split(new char[] { Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar });
+            DirectoryInfo currentDir = new DirectoryInfo(gitRepoRootDir);
+            for (int i = 0; i < segments.Length; i++)
+            {
+                string segment = segments[i];
+                var next = currentDir.GetFileSystemInfos(segment).FirstOrDefault();
```

Comment:
Ouch. I am embarrassed to have made that mistake. Nice catch. 
I've added a log.error call and then return `path` in this case.